FrameworksAndDrivers/Web/DotLanches.Api/Dtos/PagamentoResponseDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DotLanches.Api.Dtos;
+
+public class PagamentoResponseDto
```

Comment:
O nome response indica que seria uma response da nossa api, e na verdade esses são os dados de entrada do webhook, não?

Acredito que como fazemos uma decisão baseado no sucesso da operação, esse endpoint poderia ser o de processar pagamento, então esse objeto poderia ser:
```suggestion
public class ProcessPagamentoRequestDto
```